File: atividade_5/C/maioridade.c
#include <stdio.h>

int main()
{
    int idade = 0;
    printf("Digite sua idade e verifique se eh maior de idade: ");
    scanf("%u", &idade);
    fflush(stdin);
    printf(idade < 18 ? "Menor de idade" : "Maior de idade");
    return 0;
};

File: atividade_5/java/src/Maioridade.java
import java.util.Scanner;

public class Maioridade {
    public static void main(String[] args) {
        try (var scan = new Scanner(System.in)) {
            System.out.print("Digite sua idade e verifique se é maior de idade: ");
            int idade = scan.nextInt();
            System.out.println(idade < 18 ? "Menor de idade" : "Maior de idade");
        } catch (Exception e) {
            System.out.println(e.getMessage());
        }
    }
}